src/control/server/instance_storage_rpc.go

```diff
@@ -89,13 +89,7 @@ func (ei *EngineInstance) bdevFormat(p *bdev.Provider) (results proto.NvmeContro
 	ei.log.Infof("Instance %d: starting format of %s block devices %v",
 		engineIdx, cfg.Class, cfg.DeviceList)
 
-	res, err := p.Format(bdev.FormatRequest{
-		OutputPath: cfg.OutputPath,
-		Class:      cfg.Class,
-		DeviceList: cfg.DeviceList,
-		FileSize:   cfg.FileSize,
-		MemSize:    cfg.MemSize,
-	})
+	res, err := p.Format(bdev.FormatRequestFromConfig(&cfg, hostname()))
```

Comment:
This is adding a new dependency on the bdev package from the server package, but I'm OK with it for now because it should be simple to change it to `storage.BdevFormatRequestFromConfig()` once that work lands.